lib/types.nix

```diff
@@ -605,6 +609,13 @@ let
         };
       };
 
+      fileset = mkOptionType {
+        name = "fileset";
+        description = "fileset";
```

Comment:
```suggestion
        description = "fileset";
        descriptionClass = "noun";
```

Reply:
Done in e01e924f17a02d85e52c809d92792a3ea19bf44b.